server/views/pages/error.njk

```diff
@@ -9,32 +9,30 @@
 
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-xl govuk-!-margin-bottom-6" data-qa="error-heading">
+                <h1 class="govuk-heading-xl" data-qa="error-heading">
                     {% if status === 404 and not errorKey %}
                         Page not found
-                        {% elif errorKey === 'NO_CALCULATION_SUBMITTED' %}
+                    {% elif errorKey === 'NO_CALCULATION_SUBMITTED' %}
                         No calculation submitted
+                    {% elif errorKey === 'NOT_IN_CASELOAD' %}
+                        The details for this person cannot be found.
                     {% else %}
                         There is a problem
                     {% endif %}
                 </h1>
-
+            </div>
+        </div>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
                 {% if errorKey %}
                     {% if errorKey === 'NOT_IN_CASELOAD' %}
-                        <p class="govuk-body">The details for this person cannot be found.</p>
-                        <p class="govuk-body"> This could be because this person:</p>
+                        <p class="govuk-body">This could be because this person:</p>
                         <ul class="govuk-list govuk-list--bullet">
-                            <li>is not in your caseload</li>
+                            <li>is not in your case load</li>
                             <li>has no bookings assigned to them</li>
                             <li>has no active bookings</li>
```

Comment:
Are these other scenarios still relevant if we're catching & handling the specific NOT_IN_CASELOAD key?

Reply:
https://github.com/ministryofjustice/calculate-release-dates/blob/main/server/services/prisonerService.ts#L43 is the line which handles it, I think the `has no` lines would be dictated by the prisoner agency ID being OUT

Reply:
right so they're not in your case load because they're not in prison? Seems like we could handle that differently but outside the scope of this ticket.